.rubocop.yml

```diff
@@ -55,3 +52,18 @@ Style/StringLiterals:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
+Style/StringLiterals:
+  Enabled: false
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Metrics/MethodLength:
+  Enabled: false
+
+Lint/AssignmentInCondition:
+  Enabled: false
```

Comment:
I'm willing to give this one up (we don't pass on it anyways), but should we remove the previous definition from above ?